win32/tests/string_utils_win32_ut/CMakeLists.txt

```diff
@@ -17,4 +17,4 @@ set(${theseTestsName}_h_files
 mocks.h
 )
 
-build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal_reals)
+build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal_reals pal_ll_interfaces pal_ll_win32 win32_ll_reals)

```

Comment:
>pal_ll_win32 [](http://example.com/codeflow?start=120&length=12)

can we also remove this because it's a UT?

Reply:
removed